asyncscope.md

```diff
@@ -1820,8 +1820,25 @@ Specification
 ============
 
 ## `execution::async_scope_token`
+::: add
+__§33.4 Async scope token concept[asyncscopetoken.concept]__
 
-spec here
+[1]{.pnum} Let Sender be `decltype(sndr)` and Token be an async scope token type. If the type `async_scope_token<Token, Sender>` is valid, it denotes a non-owning handle to an [async scope](#executionasync_scope) that can `nest` Senders onto the token's `async_scope`.
+```cpp
+template <class Token, class Sender>
+concept async_scope_token = // exposition only
```

Comment:
Add blurb about the challenges with defining this concept and ask LEWG which design is preferred